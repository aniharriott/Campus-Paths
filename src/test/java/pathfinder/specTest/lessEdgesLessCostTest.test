# Test that findMinPath will return the shortest cost path

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

AddEdge graph1 n1 n2 6.000
AddEdge graph1 n2 n3 7.000
AddEdge graph1 n1 n3 1.000

FindPath graph1 n1 n3