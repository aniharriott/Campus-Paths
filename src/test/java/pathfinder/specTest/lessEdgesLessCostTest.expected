# Test that findMinPath will return the shortest cost path

created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

added edge 6.000 from n1 to n2 in graph1
added edge 7.000 from n2 to n3 in graph1
added edge 1.000 from n1 to n3 in graph1

path from n1 to n3:
n1 to n3 with weight 1.000
total cost: 1.000